rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/compact/Tile.java

```diff
@@ -181,13 +188,13 @@ public void onBrowserEvent(Event event) {
         }
     }
 
-    private void disableItems(MenuItem[] items) {
+    private void disableItems(MenuItem... items) {
         for (MenuItem item : items) {
             item.setEnabled(false);
         }
     }
 
-    private void enableItems(MenuItem[] items) {
+    private void enableItems(MenuItem... items) {
         for (MenuItem item : items) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "white" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)